staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -104,8 +104,8 @@ func newETCD3Client(c storagebackend.Config) (*clientv3.Client, error) {
 		Endpoints: c.ServerList,
 		TLS:       tlsConfig,
 	}
-	client, err := clientv3.New(cfg)
-	return client, err
```

Comment:
> minor fix

hi @gorilla001 thx for contributing. whats the fix btw? didn't really understand.

Reply:
there is no fix, its a minor style correction, but nonetheless a good one.

Reply:
@yue9944882 yes, its only a style correction, not a fix. @krmayankk